endpoints.tf

```diff
@@ -68,3 +68,18 @@ data "template_file" "local_metastores_yaml" {
     mapped_databases = "${ lookup(var.local_metastores[count.index],"mapped-databases","") }"
   }
 }
+
+resource "aws_route53_zone" "remote_metastore" {
```

Comment:
We cannot reuse the existing route53 zone?

Reply:
why do we need to use existing zones? these records will be used in special case and only be used inside VPC.
Existing zones may be targeted for different use.